src/fsharp/vs/Symbols.fs

```diff
@@ -805,7 +805,10 @@ and FSharpAccessibility(a:Accessibility, ?isProtected) =
 
     member __.Contents = a
 
-    override x.ToString() = stringOfAccess a
+    override x.ToString() = 
+        let (TAccess paths) = x
```

Comment:
Is this change intended (I guess helpful in debugger)? maybe a comment with what is the string looking like?